packages/nextra/src/compile.ts

```diff
@@ -21,41 +20,30 @@ const createCompiler = (mdxOptions: ProcessorOptions) => {
 
 const rehypePrettyCodeOptions = {
   theme,
-  // onVisitLine(node: any) {
-  //   // Style a line node.
-  //   Object.assign(node.style, {
-  //   })
-  // },
-  onVisitHighlightedLine(node: any) {
-    // Style a highlighted line node.
-    if (!node.properties.className) {
-      node.properties.className = []
+  onVisitLine(node) {
+    // Prevent lines from collapsing in `display: grid` mode, and
+    // allow empty lines to be copy/pasted
+    if (node.children.length === 0) {
+      node.children = [{ type: 'text', value: ' ' }]
```

Comment:
according to docs 
<img width="681" alt="image" src="https://user-images.githubusercontent.com/7361780/177035538-2b5b70c7-2b50-428b-8198-9c71a3a1cc58.png">